docs/user/basic-concepts.rst

```diff
@@ -241,6 +243,13 @@ objects are defined and managed by super administrators and can include
 configurations, policies, or other data that need to be consistent across
 all organizations.
 
+Shared objects can only be created by superusers. Non-superusers (e.g.
+:ref:`users_organization_manager`) have view-only access to shared
+objects. However, they can use these shared objects when creating related
+organization-specific resources. E.g., an organization manager can use a
+shared VPN server to create a configuration template for their
+organization.
```

Comment:
Let's mention this works both in admin and REST API, let's mention that non superusers may be prevented to viewing sensitive details of certain shared objects which would allow them to gain unauthorized access to resources used by other organizations.